meta-resin-common/recipes-support/hostapp-update-hooks/files/99-resin-grub

```diff
@@ -6,7 +6,7 @@
 
 set -o errexit
 
-new_part=$(findmnt --noheadings --canonicalize --output SOURCE "/mnt/sysroot/inactive")
```

Comment:
Here you can use the `DURING_UPDATE` variable to select the new partition

Reply:
Done. Will push with next iteration.